### PR TITLE
Support WOFF 2.0

### PR DESCRIPTION
**Pull request by htanjo**
[WOFF 2.0](http://www.w3.org/TR/WOFF2/) format is superior to WOFF, and `gulp-iconfont` supports it now.
I added `woff2` to the templates.

And I found the master branch [failed the test](https://travis-ci.org/htanjo/gulp-iconfont-css/builds/97166916), so I also updated test assets to fix it.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
 .DS_Store
 node_modules
-test/results
+test/results*
```
Comment (backflip) [inline .gitignore]: What's the reason for this change?

**Comment by backflip** [comment]
Thanks a lot, LGTM. The only question I have is with regards to the `.gitignore`

File: .gitignore (hunk 1 above)
Comment (htanjo) [inline .gitignore]: Your test code doesn't clean up `results_css`, `results_less` and `results_scss` when test failed. And `test/results` in .gitignore doesn't match these directories. 

I think test should always be passed, but it's safer to ignore `results*` directories completely.

**Comment by backflip** [comment]
Sorry about the delay, thanks again!

**Comment by htanjo** [comment]
No problem! Thanks.
Could you publish to npm if you are ready?

**Comment by backflip** [comment]
Finally published as 2.1.0: https://github.com/backflip/gulp-iconfont-css/releases

**Comment by htanjo** [comment]
Thank you :smile:
